content/prompts/ai-development/responsible-ai-checker.md

```diff
@@ -0,0 +1,50 @@
+---
+authors:
+  - lvzhaobo
+categories:
+  - ai-development
+description:
+  Validate AI applications against responsible AI principles and ethical guidelines
+draft: false
+howToUse:
+  "1. Add your AI application code to context using @workspace
+  2. Copy-paste the prompt into your chat
+  3. Review the generated assessment and implement recommendations"
+images: []
+tags:
+  - AI
+  - Ethics
+  - Security
+  - Compliance
+title: Responsible AI Checker
+createdAt: 2025-01-20T12:00:00.000Z
+updatedAt: 2025-01-20T12:00:00.000Z
+aliases: ["/prompts/prompt/responsible-ai-checker-4f8a9b2c"]
```

Comment:
Can you remove the alias? This is used to configure redirects. I used alias for the migration from the previous promptz database. However for new content this is not required. I will also check the contribution instructions and templates if there is an error in the documentation